helm/values.yaml

```diff
@@ -15,25 +15,33 @@ deployment:
   tolerations: []
   affinity: {}
 
-# configure connaisseur service
+# configure Connaisseur service
 service:
   type: ClusterIP
   port: 443
 
-# configure access to the notary server
+### VALIDATORS ###
+# validators are a set of configurations (hosts, types, public keys) that can be
+# used for validating one or multiple images (or image signatures). They are tied
+# to their respective image(s) via the image policy below. There are a few handy
+# validators pre-configured.
 validators:
+# static validator that allows each image
 - name: allow
   type: static
   approve: true
+# static validator that denies each image
 - name: deny
   type: static
   approve: false
-- name: default  # the default validator which is used, if no validators are specified
+# the `default` validator is used if no validator is specified in image policy
+- name: default
   type: notaryv1  # cosign is also experimetally supported
-  host: notary.docker.io
+  host: notary.docker.io # configure the notary server to be used
   pub_keys:
-  - name: default  # the default key which is used, if no keys are specified
-    key: |
+  # the `default` key is used if no key is specified in image policy
+  - name: default
+    key: |  # enter your key below
```

Comment:
This way the default validator is defined but not operable. Would either comment it out completly or set to docker_official

Reply:
as discussed, with @peterthomassen and @phbelitz , I switched to commenting the whole key, but key the validator